themes/bootstrap3/templates/RecordTab/holdingsils.phtml

```diff
@@ -43,7 +43,7 @@
       </div>
     <?php elseif (!$user->cat_username): ?>
       <div class="alert alert-info">
-        <?=$this->translate('hold_profile_html', ['%%url%%' => $this->escapeHtmlAttr($this->recordLinker()->getTabUrl($this->driver, 'Holdings', ['catalogLogin' => 'true']))])?>
+        <?=str_replace('<a ', '<a data-lightbox ', $this->translate('hold_profile_html', ['%%url%%' => $this->escapeHtmlAttr($this->recordLinker()->getTabUrl($this->driver, 'Holdings', ['catalogLogin' => 'true']))]))?>
```

Comment:
It seemed cleaner to adjust the link dynamically in the template instead of putting more complex markup into all the language files, but I'm open to a different approach if others prefer.